src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api.UnitTests/Controllers/ApprenticeshipUpdateControllerTests/GetApprenticeshipUpdatesTests.cs

```diff
@@ -0,0 +1,150 @@
+                Mediator.Setup(x => x.Send(It.IsAny<GetApprenticeshipUpdateQuery>(), It.IsAny<CancellationToken>()))
+                    .ReturnsAsync(QueryResult);
+
+                MapperResult = AutoFixture.Create<GetApprenticeshipUpdatesResponse>();
+
+                ModelMapper = new Mock<IModelMapper>();
+                ModelMapper.Setup(x => x.Map<GetApprenticeshipUpdatesResponse>(It.Is<GetApprenticeshipUpdateQueryResult>(r => r == QueryResult)))
+                    .ReturnsAsync(MapperResult);
+
+                ApprenticeshipId = AutoFixture.Create<long>();
+                Status = CommitmentsV2.Types.ApprenticeshipUpdateStatus.Pending;
+                Controller = new ApprenticeshipUpdateController(Mediator.Object, ModelMapper.Object);
+            }
+
+            public GetApprenticeshipUpdatesTestsFixture WithNoPendingUpdateFound()
+            {
+                QueryResult = new GetApprenticeshipUpdateQueryResult { ApprenticeshipUpdates = new List<GetApprenticeshipUpdateQueryResult.ApprenticeshipUpdate>() };
+                Mediator.Setup(x => x.Send(It.IsAny<GetApprenticeshipUpdateQuery>(), It.IsAny<CancellationToken>()))
+                    .ReturnsAsync(QueryResult);
+
+                MapperResult = new GetApprenticeshipUpdatesResponse { ApprenticeshipUpdates = new List<GetApprenticeshipUpdatesResponse.ApprenticeshipUpdate>() };
+                ModelMapper.Setup(x => x.Map<GetApprenticeshipUpdatesResponse>(It.Is<GetApprenticeshipUpdateQueryResult>(r => r == QueryResult)))
+                   .ReturnsAsync(MapperResult);
+
+                return this;
+            }
+
+            internal GetApprenticeshipUpdatesTestsFixture SetStatus(ApprenticeshipUpdateStatus? status)
+            {
+                Status = status;
+                return this;
+            }
+
+            public async Task GetApprenticeshipUpdates()
+            {
+                Result = await Controller.GetApprenticeshipUpdates(ApprenticeshipId, Status);
+            }
+
+            public void VerifyApprenticeshipIdAndStatusPassedToQuery()
+            {
+                Mediator.Verify(x => x.Send(It.Is<GetApprenticeshipUpdateQuery>(p => p.ApprenticeshipId == ApprenticeshipId && p.Status == Status ), It.IsAny<CancellationToken>()));
+            }
+
+            public void VerifyResult()
+            {
+                Assert.IsInstanceOf<OkObjectResult>(Result);
+                var resultObject = (OkObjectResult)Result;
+                Assert.IsInstanceOf<GetApprenticeshipUpdatesResponse>(resultObject.Value);
+                Assert.AreSame(MapperResult, resultObject.Value);
+            }
+
+            public void VerifyNotFoundResult()
+            {
+                Assert.IsNotNull(Result);
+                var model = Result.VerifyReturnsModel().WithModel<GetApprenticeshipUpdatesResponse>();
+                Assert.AreEqual(0, model.ApprenticeshipUpdates.Count);
+            }
+        }
+    }
+}
+
+
```

Comment:
Paul will freak out if he sees all this whitespace...

Reply:
:) removed.